src/genie/libs/parser/nxos/show_vrf.py

```diff
@@ -292,7 +292,7 @@ def cli(self, vrf=None):
         #     route-target both auto
         #     route-target both auto mvpn
         #     route-target both auto evpn
-        p5 = re.compile(r'^\s*route-target +(?P<rt_type>\w+) +(?P<rt>\w+)( +(?P<rt_evpn_mvpn>\w+))?$')
+        p5 = re.compile(r'^\s*route-target +(?P<rt_type>\w+) +(?P<rt>\w+:\w+)( +(?P<rt_evpn_mvpn>\w+))?$')
```

Comment:
Please add comment for the regex change

Reply:
Sorry, but I'm not sure what to add since it should have been like this from the start.